composer.json

```diff
@@ -51,5 +50,12 @@
         "lint-clean": "phpcbf src/ tests/php/"
     },
     "prefer-stable": true,
-    "minimum-stability": "dev"
+    "minimum-stability": "dev",
+    "config": {
+        "allow-plugins": {
+            "composer/installers": true,
+            "silverstripe/recipe-plugin": true,
+            "silverstripe/vendor-plugin": true
+        }
+    }
```

Comment:
Please remove the branch-alias from composer.json

I've created new `5` and `5.1` branches from `master` and retargetted this PR to `5`

Please also create a follow up PR to remove the branch alias from the `master` branch

You'll want to check relevant projects that they're requiring this as `^5` rather than `dev-master`

Reply:
I don't really understand what the consequences of those changes are, but they kinda sound like general house keeping? Does this all need to part of this PHP 8 upgrade PR?

Reply:
Preferably now .. we're targeting this at the `5` branch (which should have always existed), so if we keep the existing branch-alias's in the `5` and `master` branches there' a chance that requesting `5.x-dev` will mistakenly fetch from the `master branch` when it shouldn't